### PR TITLE
Adds pycryptodome tests

### PR DESCRIPTION
**Pull request by KaiSforza**
Removes the m2crypto and pycrypto packages and installs the required `pycryptodome{,x}` package. I set the version to the current latest or above, let me know if we can change that.

@gtmanfred @dubb-b

### CODE REVIEW

**Comment by gtmanfred** [comment]
I do not believe we are testing pycryptodome, we will only be using pycryptodomeex, so that we do not conflict with peoples usage of pycrypto.

File: python/pycryptodomex.sls [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+include:
+  - gcc
+  - python.pip
+{% endif %}
+
+{% for cpkg in ['m2crypto', 'pycrpyto'] %}
+remove-{{cpkg}}:
+  pip.removed:
+    - name: {{cpkg}}
+{% endfor %}
```
Comment (gtmanfred) [inline python/pycryptodomex.sls]: This is also not needed, we don't need to remove pycrypto, and pycrypto should be spelled correctly :P

**Comment by rallytime** [comment]
Where are we with this @KaiSforza?

File: python/pycryptodomex.sls (hunk 1 above)
Comment (KaiSforza) [reply python/pycryptodomex.sls]: I'm going to keep the removal in there, just to make sure we don't have any packages we don't want.

**Comment by rallytime** [comment]
@KaiSforza Just checking in here with this change. What's the plan with this PR?

**Pull request closed without merging**